components/Cards/examples/EditReorderCollectionViewController.swift

```diff
@@ -60,8 +60,13 @@ class EditReorderCollectionViewController: UIViewController,
       collectionView.addGestureRecognizer(longPressGesture)
     }
 
+    let titles = ["Mainsail", "Break point", "Matchday", "Blue run", "90th Minute",
+                  "Race track", "Backcourt", "Alley-oop", "Approach", "Scoring",
+                  "Master point", "Overhead", "Relay", "Tie breaker", "Break shot",
+                  "Track race", "Wide ball", "Sideback", "Baseline", "Off side"]
+
     for i in 0..<20 {
```

Comment:
Maybe use `titles.count` instead of hard-coding 20?

Reply:
sure